services/worker.go

```diff
@@ -0,0 +1,34 @@
+package services
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/rke/hosts"
+)
+
+func RunWorkerPlane(masterHosts []hosts.Host, workerHosts []hosts.Host, workerServices Services) error {
+	logrus.Infof("[WorkerPlane] Building up Worker Plane..")
+	for _, host := range masterHosts {
+		// only one master for now
+		err := runKubelet(host, masterHosts[0], workerServices.Kubelet, true)
```

Comment:
Why do we run kubelet on master hosts? Isn't it supposed to run on worker hosts only?

Reply:
No kubelet need to run on master hosts as well, however it should be in non schedulable state, the node still should be monitored by kubernetes